docs/source/Plugin/P014.rst

```diff
@@ -57,9 +57,9 @@ Do not use the single ended circuit as the Vout is connected directly to ground:
 
 
 **Note:**
-The plugin reads the CHIP ID to determine if the ADC is available.
+The plugin reads the CHIP ID to determine if the ADC is available. This Plugin is NOT compatible with STH20, for SHT2x devices use Plugin 122.

```

Comment:
A typo crept in there STH instead of SHT, and you might want to name P122 by description, or add a link to the documentation.

Reply:
Typo corrected. Thanks!